python/audit/tests/test_audit_objects.py

```diff
@@ -0,0 +1,153 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+from django.urls import reverse
+from rest_framework.response import Response
+from rest_framework.status import HTTP_201_CREATED, HTTP_204_NO_CONTENT
+
+from adcm.tests.base import BaseTestCase
+from audit.models import AuditObject, AuditObjectType
+from cm.models import Bundle, Prototype, PrototypeConfig
+
+
+class TestAuditObjects(BaseTestCase):
+
+    cluster_proto: Prototype
```

Comment:
I think it's better to get rid of the class variables, and transfer them to a `setUp()` or a function `_prepare_prototypes()`

Reply:
Moved to `setUp` with assignment of `None` value to help IDE in type hints